src/arch/armv8/armv8-a/psci.c

```diff
@@ -58,7 +58,7 @@ static void psci_wake_from_powerdown(void)
 
 static void psci_wake_from_idle(void)
 {
-    cpu_idle_wakeup();
+    cpu_standby_wakeup();
```

Comment:
We can change this "idle" nomenclature to "standby"

Reply:
Done [here](https://github.com/bao-project/bao-hypervisor/pull/186/commits/1dba63da1c20589c1bafba8bdf6575df578b2fcb)